charts/astronomer/values.yaml

```
@@ -210,8 +210,8 @@ houston:
   # Cleanup task usage data and task audit data in Houston
   # This runs as a CronJob
   cleanupTaskUsageData:
-    # Run at 23:40 every night
-    schedule: "40 23 * * *"  # https://crontab.guru/#40_23_*_*_*
+    # Run at 23:00 every night
+    schedule: "00 23 * * *"  # https://crontab.guru/#00_23_*_*_*
```

We should avoid minute 0 with all our cronjobs. Minute 0 is a hot time for job schedules, and adding more to that will make that minute even more resource demanding. Using a different minute will smooth resource demand out over time.

```suggestion
    # Run at 23:11 every night
    schedule: "11 23 * * *"  # https://crontab.guru/#11_23_*_*_*
```

---

We do not appear to have test cases for cleanupTaskUsageData defaults or custom entries.